components/ContentSearch/index.js

```diff
@@ -5,7 +5,9 @@ import PropTypes from 'prop-types';
 import { __ } from '@wordpress/i18n';
 import SearchItem from './SearchItem';
 
-const NAMESPACE = '10up-block-components';
+const NAMESPACE = '10up-content-search';
+
+const searchCache = {};
```

Comment:
This should probably be state within the component, because if it's at the module level, then multiple instances of the component will end up using the same cache object, even though they may have different contentTypes. Alternatively, we could key this cache object by the full search URL (or JSON.stringified query args) instead of just the keyword.